src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -27,6 +32,8 @@
 
     private static final Logger log = LoggerFactory.getLogger(Application.class);
 
+    private static final int HEALTH_SERVER_PORT = 8081;
```

Comment:
Since the HTTP port is configurable, I think this should be configurable as well and might need to prevent conflicts with the regular port.

Reply:
I guess env var is enough here, so it's something that @stanlyDoge has to take into account for his CO integration.

Reply:
Yeah ... unfortunately ... this is why I suggested to not have the HTTP port configurable. Now you will need to deal with conflicts, make them both configurable etc. :-(

Reply:
@scholzj now the health port is got from env var and it checks conflicts with AMQP and HTTP ports exiting the application.